docs/agent-config.md

```diff
@@ -52,6 +52,16 @@ You can provide a path to agent configuration file by setting the corresponding
 |--------------------------------------|--------------------------------------|----------------------------------------------------------------------------------|
 | `otel.javaagent.configuration-file` | `OTEL_JAVAAGENT_CONFIGURATION_FILE` | Path to valid Java properties file which contains the javaagent configuration.|
 
+### Extensions
+
+You can enable [extensions](../examples/extension/README.md) as separate files by setting the corresponding property.
```

Comment:
"as separate files" sounds strange to me. I don't quite understand what does it mean.

Reply:
You can set more than extension using the same value, right?

Reply:
Fixed